runtime/src/snapshot_hash.rs

```diff
@@ -0,0 +1,5 @@
+//! Helper types and functions for handling and dealing with snapshot hashes.
+use solana_sdk::{clock::Slot, hash::Hash};
+
+type SlotHash = (Slot, Hash);
+pub type SnapshotHash = SlotHash;
```

Comment:
imo this is `type` alias abuse. `(Slot, Hash)` is a pretty simple type and I personally prefer to see the explicit type rather than some type-alias, which I have to go lookup elsewhere what that is.
I feel like this actually makes the code less readable by moving the definition further from the uses.

Reply:
Understood. Will close.